ert/data/record/_transformation.py

```diff
@@ -83,6 +160,18 @@ async def transform_input(
         path.chmod(st.st_mode | stat.S_IEXEC)
 
     async def transform_output(self, mime: str, location: Path) -> Record:
+        """Transforms a file to Record from the given location via
+        a serializer given in the mime type..
+
+        Args:
+            mime: mime type to fetch the corresponding serializer
+            runpath: the root of the path
+            location: filename to load the Record from
+
+        Returns:
+            Record: return object is either :class:`BlobRecord` or
+                :class:`NumericalRecord`
```

Comment:
Can the Executable transform really return a numerical record?

Reply:
Maybe not, at least it doesn't make sense. Actually this transformation should be used only for preparing the input; ie. `transform_input` functon.